src/enrichers/people-verified.js

```diff
@@ -1,12 +1,11 @@
-import * as error from '../utils/emitter'
 import { PEOPLE_VERIFIED_LS_ENTRY } from '../utils/consts'
 
-export function enrich (state, storageHandler) {
+export function enrich (state, storageHandler, messageBus) {
   console.log('people-verified.enrich', state)
   try {
     return { peopleVerifiedId: state.peopleVerifiedId || storageHandler.getDataFromLocalStorage(PEOPLE_VERIFIED_LS_ENTRY) }
   } catch (e) {
-    error.error('PeopleVerifiedEnrich', e.message, e)
+    messageBus.emitError('PeopleVerifiedEnrich', e.message, e)
```

Comment:
We actually use `encodeEmitError` here. We then would not need to call `e.message` here, the function would do it for us.